lib/publiccloud/provider.pm

```diff
@@ -19,7 +19,7 @@ use Data::Dumper;
 use Mojo::JSON 'decode_json';
 
 use constant TERRAFORM_DIR     => '/root/terraform';
-use constant TERRAFORM_TIMEOUT => 600;
+use constant TERRAFORM_TIMEOUT => 1000;
```

Comment:
what about 
```suggestion
use constant TERRAFORM_TIMEOUT => 17 * 60;
```
TO get a better idea of how long it is?

Reply:
done